src/alerts/generate_signals.R

```diff
@@ -179,7 +179,10 @@ generate_signals <- function(
     df_campaigns,
     fn_signals
   )
+
   log_info(paste0("Monitoring completed. ", nrow(df_campaigns), " signals generated for ", indicator_id))
+
+  df_campaigns
```

Comment:
Ahh interesting! I didn't know that the final line in a function was what was returned. I actually removed this return as it was adding a bit more clutter to the logs. Is there anywhere that this function needs to return anything, technically speaking?

Reply:
Technically it doesn't, but it's nice when developing to be able to output the data frame and explore it. I think if it's cluttering the logs, the answer in R is always to assign the variable to something. So in the `update_{indicator}.R` scripts, we can make sure `generate_signals()` is assigned to an object, e.g. `df_blah <- generate_signals()`.

Reply:
@hannahker Have pushed it so the `generate_signals()` calls are saved to a var. Let's see if that clears up the logs? Is there a way I can test before merging?

Reply:
Yup you can run a workflow off of a branch! 
![Screenshot 2024-05-28 at 12 29 46 PM](https://github.com/OCHA-DAP/hdx-signals/assets/25990216/642b9627-9e92-4cc4-b9c2-46695019cd72)